tfe/provider.go

```diff
@@ -16,10 +17,9 @@ import (
 	"github.com/hashicorp/terraform/terraform"
 )
 
-const (
-	defaultHostname = "app.terraform.io"
-	serviceID       = "tfe.v2"
-)
+const defaultHostname = "app.terraform.io"
+
+var serviceIDs = []string{"tfe.v2", "tfe.v2.1"}
```

Comment:
given the function below pulls out the first value found, do we not want to start with the newest supported and work back?